OG/OG/AST/AntlrToProgramAST.cs

```diff
@@ -2,10 +2,10 @@
 using System.Collections.Generic;
 using OG.AST;
 using OG.AST.MachineSettings;
-namespace OG.gen
+namespace OG.AST
 
 {
-    public class AntlrToProgramAST:OGBaseVisitor<ASTNode>, ISemanticErrorable
+    internal class AntlrToProgramAST:OGBaseVisitor<OGProgramAST>, ISemanticErrorable
```

Comment:
Hvad er argumentet for "internal"? Ville ikke lave det hele om til public før jeg havde hørt dit argument ;) 

Tænker det giver problemer med at bruge de her objekter i testene? Men er ikke super skarp på det keyword xD